packages/components/src/PlaceCard/PlaceCard.stories.js

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
Could we maybe just delete this file?

Though storybook is nice to have, we wanted fast iterations for this project, and storybook tends to make development a little bit slower.

Reply:
Also, you should be able to retrieve this file from universal-components git history if need be later.

Reply:
I thought about it, and I agree with you.
It's better don't add dead or commented code in project.
It should be there code only what we use, mostly.